.changeset/heavy-ducks-leave.md

```diff
@@ -0,0 +1,5 @@
+---
+"svelte": patch
+---
+
+fix: robustify interop of exports and props
```

Comment:
should this be considered a breaking change?

```suggestion
breaking: robustify interop of exports and props
```

Reply:
Mhm true - in runes mode at least this is a breaking change, so I should also note that in the docs